.github/workflows/deploy-staging.yml

```
@@ -0,0 +1,77 @@
+          git tag $TAG
+          git push origin $TAG
+
+      - name: 📣 Notify Slack on Success
+        if: success()
+        run: |
+          curl -X POST --data-urlencode "payload={\"channel\": \"#déploiement-ios\", \"username\": \"deploy-bot\", \"text\": \"🚀 Déploiement réussi de la v${VERSION} sur le canal staging. <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|Voir les logs>\", \"icon_emoji\": \":rocket:\"}" https://hooks.slack.com/services/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
+
+      - name: 📣 Notify Slack on Failure
+        if: failure()
+        run: |
+          curl -X POST --data-urlencode "payload={\"channel\": \"#déploiement-ios\", \"username\": \"deploy-bot\", \"text\": \"💣 Échec du déploiement de la v${VERSION} sur le canal staging. <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|Voir les logs>\", \"icon_emoji\": \":boom:\"}" https://hooks.slack.com/services/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
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)